tests/test_qpu.py

```diff
@@ -66,7 +64,6 @@ def test_qpu_args(self):
         with pytest.raises(ValueError, match="Device has a fixed number of"):
             qml.device("forest.qpu", device=device, shots=5, wires=range(100), load_qc=False)
 
-    @flaky(max_runs=10, min_passes=3)
```

Comment:
Do we need to remove flaky here? Is it to stop 10 repetitions of a test we know will fail?

Reply:
Yes, that was the original purpose, but rather placed them in a comment now.